src/cur-mot/add-who.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+"""
+Add `who=` attribute to signators.
+"""
+from common.xml_utils import write_xml
+from glob import glob
+from lxml import etree
+from pyriksdagen.utils import (
+    parse_protocol
+)
+from tqdm import tqdm
+import argparse, os, sys
+
+
+
+
+def get_person_D(root, ns):
+    D = {}
+    person_list = root.find(f".//{ns['tei_ns']}listPerson")
+    for p in person_list:
+        try:
+            _name = p.find(f"{ns['tei_ns']}name").text.strip()
+        except:
+            _name = None
+        try:
+            idno = p.find(f"{ns['tei_ns']}idno").text.strip()
+        except:
+            idno = None
+        if idno is not None and _name is not None:
+            D[idno] = _name
+    return D
+
+
+def match_ids(root, ns, person_D, unmatched, start, end):
```

Comment:
We have quite well-optimized fuzzy matching on the records size, why are we not using that?